docs/tutorial/part-four/index.md

```diff
@@ -354,26 +354,26 @@ export default ({ children }) => {
 // highlight-end
 ```
 
-Another success! 🎉
+Erneut ein Erfolg! 🎉
 
-![Page title and layout title both pulling from siteMetadata](site-metadata-two-titles.png)
+![Seitentitel und Layouttitel beziehen nun beide den Wert von siteMetadata](site-metadata-two-titles.png)
 
-Why use two different queries here? These examples were quick introductions to
-the query types, how they are formatted, and where they can be used. For now,
-keep in mind that only pages can make page queries. Non-page components, such as
-Layout, can use StaticQuery. [Part 7](/tutorial/part-seven/) of the tutorial explains these in greater
-depth.
+Warum nutzen wir zwei unterschiedliche Abfragen hier? Diese Beispiele waren eine
+schnelle Einführung zu den Abfragentypen, wie sie formattiert werden
+und wo sie genutzt werden. Für den Augenblick, solltest du beachten,
+dass Seitenabfragen nur von Seiten durchgeführt werden können. Komponenten die
+keine Seite darstellen, wie Layout, können StaticQuery nutzen. Im [Teil 7](/tutorial/part-seven/) des Tutorials
+werden diese ausführlicher behandelt.
 
-But let's restore the real title.
+Lass uns jedoch den echten Seitentitel wiederherstellen.
 
-One of the core principles of Gatsby is that _creators need an immediate connection to what they're creating_ ([hat tip to Bret Victor](http://blog.ezyang.com/2012/02/transcript-of-inventing-on-principle/)). In other words, when you make any change to code you should immediately see the effect of that change. You manipulate an input of Gatsby and you see the new output showing up on the screen.
+Eines der Grundprinzipien von Gatsby ist, dass *Erschaffer eine unmittelbare Verbindung zu ihrer Kreation brauchen* ([Kudos an Bret Victor](http://blog.ezyang.com/2012/02/transcript-of-inventing-on-principle/)). In anderen Worten, wenn du eine Änderung in deinem Code machst, solltest du unmittelbar die Auswirkung dieser Änderung sehen. Du manipulierst den Input von Gatsby und siehst wie der neue Output auf dem Bildschirm erscheint.
 
-So almost everywhere, changes you make will immediately take effect. Edit the `gatsby-config.js` file again, this time changing the `title` back to "Pandas Eating Lots". The change should show up very quickly in your site pages.
+Dies findet fast überall statt, Änderungen die von dir durchgeführt werden, treten unmittelbar in Kraft. Bearbeite wieder die `gatsby-config.js` Datei, ändere dieses Mal den `title`zurück auf "Pandas essen viel". Die Änderung sollte sehr schnell auf den Seiten deiner Website erscheinen.
```

Comment:
Instead of "Seiten deiner Website" simply "aud deiner Webseite"?

Reply:
I changed to "...auf deiner Website", see above regarding "Website" vs "Webseite"